packages/ckeditor5-table/src/tablenavigation.js

```diff
@@ -197,17 +197,16 @@ export default class TableNavigation extends Plugin {
 		const selectedCells = getSelectedTableCells( selection );
 
 		if ( selectedCells.length ) {
-			if ( expandSelection ) {
-				const tableSelection = this.editor.plugins.get( 'TableSelection' );
-				const focusCell = tableSelection.getFocusCell();
+			let focusCell = null;
```

Comment:
```suggestion
			let focusCell;
```

I see that `null` as an overhead. Usually we do not set `null` in similar constructs.